profiles/services/peoplefinder.py

```diff
@@ -1,15 +1,32 @@
-from typing import Optional
+from typing import TYPE_CHECKING, Optional
 
+from django.contrib.admin.models import DELETION, LogEntry
+from django.contrib.admin.options import get_content_type_for_model
+from django.contrib.auth import get_user_model
+
+from profiles.models import PeopleFinderProfile
 from profiles.models.generic import Country, UkStaffLocation
 from profiles.models.peoplefinder import PeopleFinderProfile
 from profiles.types import UNSET, Unset
 
 
+if TYPE_CHECKING:
+    from user.models import User
+else:
+    User = get_user_model()
+
+
 def get_profile_completion(peoplefinder_profile):
     # TODO: Implement get_profile_completion() function.
     return 0
 
 
+def get_by_id(sso_email_id: str) -> PeopleFinderProfile:
```

Comment:
```suggestion
def get_by_id(sso_email_id: str, include_inactive: bool = False) -> PeopleFinderProfile:
```
This override needs to be added and handled (ie by default active only ought to be returned)